4-fleetscope/modules/env_baseline/mcg.tf

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  fleet_membership_re = "//gkehub.googleapis.com/(.*)$"
+}
+
+resource "google_project_service" "mci" {
```

Comment:
Do we want to do all the service activation in setup? https://github.com/GoogleCloudPlatform/terraform-google-enterprise-application/blob/main/test/setup/main.tf#L107. Might also be a question for @gtsorbo 

Otherwise you could do here with a `for_each` and depend upon the block in the `depends_on` rather than two separate `google_project_service`.

Reply:
made the change to use for_each, @gtsorbo what is your take on the question? i.e. do all the service activation in setup

Reply:
moved the services activation to fleet project creation